src/main/java/io/kamax/matrix/client/MatrixHttpRoom.java

```diff
@@ -268,4 +274,120 @@ public _MatrixRoomMessageChunk getMessages(_MatrixRoomMessageChunkOptions option
         return new MatrixRoomMessageChunk(body.getStart(), body.getEnd(),
                 body.getChunk().stream().map(MatrixJsonPersistentEvent::new).collect(Collectors.toList()));
     }
+
+    @Override
+    public List<RoomTag> getUserTags() {
+        return getAllTags().stream().filter(tag -> "u".equals(tag.getNamespace())).collect(Collectors.toList());
+    }
+
+    private List<RoomTag> getAllTags() {
+        List<RoomTag> tags = new ArrayList<>();
+
+        URIBuilder builder = getClientPathBuilder("/user/{userId}/rooms/{roomId}/tags");
+
+        String body = execute(new HttpGet(getWithAccessToken(builder)));
+
+        if (StringUtils.isNotEmpty(body)) {
+            JsonObject jsonTags = jsonParser.parse(body).getAsJsonObject().get("tags").getAsJsonObject();
```

Comment:
Let's use `GsonUtil.parseObj()` and `GsonUtil.getObj()` instead (trying to make it consistent across the code)